src/utils/strings/__tests__/comma_separated_list.js

```diff
@@ -0,0 +1,39 @@
+import expect from 'expect.js';
+
+import { parseCommaSeparatedList } from '../comma_separated_list';
+
+describe('utils parseCommaSeparatedList()', () => {
+  it('supports non-string values', () => {
```

Comment:
hopefully with `Typescript` and strict signature definition we'll be able to get rid of such not obvious "feature" :D